packages/paste-icons/tools/templates/reactIconTemplate.js

```diff
@@ -13,20 +13,22 @@ export interface ${componentName}Props extends IconWrapperProps {
   decorative: boolean;
 }
 
-const ${componentName}: React.FC<${componentName}Props> = ({as, display, element = 'ICON', size, color, title, decorative}) => {
+const ${componentName} = React.forwardRef(function ${componentName}(
```

Comment:
Note: looking at [the React docs](https://reactjs.org/docs/forwarding-refs.html#displaying-a-custom-name-in-devtools), it seems like the way to add a display name to forwardRef components is to use a named function like this:

```tsx
 const MyIcon = React.forwardRef(function MyIcon() {})
```

but when I did that, Avatar still wasn't able to get the displayName correctly. So, I ended up leaving line 33 in too.

Reply:
> but when I did that, Avatar still wasn't able to get the displayName correctly. So, I ended up leaving line 33 in too.

question: does this mean line 33 still works? I thought it wasn't working and then started to work when we made this change.